distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestApplicationService.java

```diff
@@ -128,6 +125,7 @@ public void beforeExam() throws Exception {
 
     @Test
     public void aTestStartUpWithApplistPropertiesFile() throws Exception {
+        ApplicationService applicationService = getServiceManager().getService(ApplicationService.class);
```

Comment:
I think this could return null.  Maybe we should wait for it not to be null in the beforeExam.